moveit_plugins/moveit_simple_controller_manager/include/moveit_simple_controller_manager/action_based_controller_handle.h

```diff
@@ -51,7 +51,8 @@ namespace moveit_simple_controller_manager
 class ActionBasedControllerHandleBase : public moveit_controller_manager::MoveItControllerHandle
 {
 public:
-  ActionBasedControllerHandleBase(const std::string& name) : moveit_controller_manager::MoveItControllerHandle(name)
+  ActionBasedControllerHandleBase(const std::string& name, const std::string& logger_name)
+    : moveit_controller_manager::MoveItControllerHandle(name), LOGGER(rclcpp::get_logger(logger_name))
```

Comment:
The default constructor is implicitly deleted the only public constructor is the copy constructor, the constructor with string is private with get_logger as a friend to the class